crud.py

```diff
@@ -0,0 +1,75 @@
+from sqlalchemy.orm import Session
+import models
+import schemas
+from fastapi import HTTPException
+
+
+def get_all_authors(
+        db: Session,
+        skip: int | None = None,
+        limit: int | None = None,
```

Comment:
Similar to the previous comment, ensure compatibility with Python versions below 3.10 by using `Optional[int]` for type hinting.